src/lib/comms/sol-oic-server.c

```diff
@@ -39,6 +39,12 @@
 #include "sol-oic-server.h"
 #include "sol-oic-security.h"
 
+#define STR(s) #s
+#define TO_STRING(s) STR(s)
+
+#define OIC_DATA_MODEL_VERSION "Unknown"
+#define OIC_SPEC_VERSION ""
```

Comment:
Those are information related to the oic implementation we are using. There is no need to let user change them. I'll just fix OIC_SPEC_VERSION to have the current OIC spec implemented.